Apex_Unit_Testing/Apex_Language_Test Case/Apex2 Test Case/AccountUtilTest.cls

```diff
@@ -0,0 +1,38 @@
+@isTest
+public class AccountUtilTest 
+{
+    // Test when contactId of account field is null
+    @isTest
+    static void TestAccountWhenContactIdFieldIsNull()
+    {
+        Account acct = new Account(Name = 'Test Account');
+        insert acct;
+        Contact cont = new Contact(LastName = 'Prem', AccountId = acct.Id);
+        insert cont;
+        AccountUtil.storeContactIds(cont.Id);
+        Account act = [SELECT ContactIds__c 
+                       FROM Account 
+                       WHERE Id =: acct.Id];
+        System.assertnotEquals(string.valueOf(cont.Id), act.ContactIds__c);
```

Comment:
isn't the method supposed to store the passes contact id in ContactIds__c if the field is null?